### PR TITLE
[PROPOSAL]:  PR template

### PR DESCRIPTION
**Pull request by salvar3nga**
Issue #204

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+### 🗒 Description
+
+If helpful for the review process, please describe your changes from a technical or non technical perspective
+
+#
+### 🎫 Git ticket reference
```
Comment (illia-solovei-s1) [inline .github/pull_request_template.md:6]: The ticket can be attached using measures from Github itself

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- 
+
+#
+### 📷 Demo screen recording or Screenshots
```
Comment (illia-solovei-s1) [inline .github/pull_request_template.md:11]: Most PRs don't require this (unless its soully UI changes), I wouldn't include it in the template and make it completely optional.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- [x] Not applicable
+
+# 
+### 📋 Checklist
+
+You may remove tasks that do not make sense in this PR.
+
+- [ ] GitHub **labels** are assigned
```
Comment (illia-solovei-s1) [inline .github/pull_request_template.md:20]: Its better to rearrange them in order of how often a particular check is used, e.g. you can start with checking all ACs, then move on to tested and so on.

File: .github/pull_request_template.md (hunk 1 above)
Comment (illia-solovei-s1) [reply .github/pull_request_template.md:6]: Measures in question

image

Comment (illia-solovei-s1) [reply .github/pull_request_template.md:6]: Sounds good to me, if we add this as a check

**Review by illia-solovei-s1 — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (michelguerin) [reply .github/pull_request_template.md]: Haha, I didn't see this option in the side panel. I think I do not look often on the right of the issue.
I personally perfer to link them inline as text with key words like closes or closed. So I liked this part :)

**Comment by michelguerin** [comment]
History is not always clean, and way of working not always linear. I like to add a part to explain how to start the review, by linking usefull commits or explaining how I build the feature. It gives some keys reading all the files, and maybe go through the review a little faster.
Screenshot 2024-11-25 at 11 30 11

**Comment by salvar3nga** [comment]
I'll add that section into the template as well @michelguerin

**Pull request merged**